homeassistant/components/transmission/sensor.py

```diff
@@ -35,54 +33,56 @@ async def async_setup_entry(
 ) -> None:
     """Set up the Transmission sensors."""
 
-    tm_client: TransmissionClient = hass.data[DOMAIN][config_entry.entry_id]
+    coordinator: TransmissionDataUpdateCoordinator = hass.data[DOMAIN][
+        config_entry.entry_id
+    ]
     name: str = config_entry.data[CONF_NAME]
 
     dev = [
         TransmissionSpeedSensor(
-            tm_client,
+            coordinator,
```

Comment:
In a follow-up PR it would be good to move this to `EntityDescriptions` and also add UoM, device_class, state_class etc.